### PR TITLE
Update the Custom Order Status page information

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request updates the Custom Order Status page information

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- no

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

https://docs.magento.com/user-guide/sales/order-status-custom.html

### CODE REVIEW

File: src/sales/order-status-custom.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Custom Order Status
 ---
 
-In addition to the preset order status settings, you can create custom order status settings of your own, assign them to order states, and set a default order status for order states. For example, you might need a custom order status such as “packaging”, “backordered”, or a status that is specific to your needs. You can create a descriptive name for the custom status and assign it to the associated order state in the workflow.
+In addition to the preset order status settings, you can create custom order status settings of your own, assign them to order states, and set a default order status for order states. The order state indicates the position of the order in the order processing workflow and the order status defines the state of the order. For example, you might need a custom order status such as `packaging”`, `backordered`, or a status that is specific to your needs. You can create a descriptive name for the custom status and assign it to the associated order state in the workflow.
```
Comment (leslietilling) [inline src/sales/order-status-custom.md:5]:
```suggestion
In addition to the preset order status settings, you can create your own custom order status settings, assign them to order states, and set a default order status for order states. The order state indicates the position of the order within the order processing workflow and the order status defines the state of the order. For example, you might need a custom order status such as `packaging”`, `backordered`, or a status that is specific to your needs. You can create a descriptive name for the custom status and assign it to the associated order state in the workflow.
```

File: src/sales/order-status-custom.md [hunk 2]
```diff
@@ -46,6 +46,9 @@ _Order Status Settings_
 
    - To make this status the default for the order state, select the **Use Order Status as Default** checkbox.
 
+   {:.bs-callout-info}
+   Only the default order statuses are used in the order workflow. Non-default statuses can be set only in the order comments section in the admin panel.
```
Comment (leslietilling) [inline src/sales/order-status-custom.md:50]:
```suggestion
   Only the default order statuses are used in the order workflow. Non-default statuses can be set only in the order comments section from the Admin.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
just minor edits

**Comment by ZhovnirMaria** [comment]
@leslietilling I have updated my PR, please check. Thanks ☺️

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@dobooth the Pull Request is successfully imported.

**Review by leslietilling — APPROVED** [review]

**Pull request merged**